src/components/BasicTable.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './BasicTable.scss';
+
+const getTableHeaders = (object = {}) => {
+  return Object.keys(object);
+}
+
+const generateRows = (row = {}) => {
+  return (
+    <tr>
+      {Object.values(row).map(
+        value => <td>{value}</td>
+      )}
+    </tr>
+  )
+}
+
+const BasicTable = ({ data }) => {
+  return (
+    <table className="basic-table">
+      <tr>
+        {getTableHeaders(data[0]).map(
```

Comment:
you use `get` and `generate` for two similar actions. can you use the same verb?